seismic_site/seismic_site/urls.py

```diff
@@ -15,20 +15,18 @@
 """
 from django.contrib import admin
 from django.urls import path, include
-from rest_framework.routers import DefaultRouter
-from django.conf import settings
-from django.conf.urls.static import static
 
 from . import views
-from map_app.views import BuildingViewSet
 
-router = DefaultRouter()
-router.register(r'buildings', BuildingViewSet)
+from django.conf import settings
+from django.conf.urls.static import static
 
 
 urlpatterns = [
+    # API
     path('api/v1/', include('api.urls')),  # This is the legacy API
-    path('api/v2/', include(router.urls)),
+    path('api/v2/', include(('buildings.urls', 'buildings'), namespace='buildings-api')),
```

Comment:
I don't think that we should assign the entire "api/v2/" URL to the buildings API. The site might need some API methods for the CMS too.

Reply:
I guess that we could add later 
`path('api/v2/', include(('cms.urls', 'cms'), namespace='cms-api')),`

but I think that being more explicit about the path is better. For example:
`path('api/v2/buildings/', include(('buildings.urls', 'buildings'), namespace='buildings-api')),` (and remove the `r'buildings'` from the router)

Reply:
That would mean having one global router. I opt for having all routes specific to a django app encapsulated in its own router, in the same fashion we'd do with `urls.py` files.
Further justification: when editing, you have them where you are, inside the app. If you need discovery, the DRF ui is the best way to go.

Reply:
We'll proceed with the changes @iuliux made and we'll figure out if it's the right way going forward